### ISSUE TITLE
Supply containers got a "hole" in the center? Add CLS support.

### ISSUE THREAD

**Issue by Gordon-Dry** [steamp0rt/Kerbalism#39]
I just see that I could stack them nicely, they have kinda hole in the center but no CLS support.

It would be fairly easy to add just the CLS support by patching, but also changing the texture of the top and bottom to show that they really are passable would be great.

btw I'm talking about the InlineContainers, at least the 125, 250 and 375 should get CLS support - the 0625 is too small...

Actually the 250 and 375 already got a texture on top/bottom that looks like they're passable.

**Comment by PiezPiedPy** [comment]
Personally I have no idea how to modify the parts or textures, could you give details how to add the CLS support and I will include it.

**Comment by Gordon-Dry** [comment]
New config, when I'm right just take it.
GameData\Kerbalism\Support\CLS.cfg
```
@PART[kerbalism-greenhouse]:HAS[!MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]
{
  MODULE
  {
    name = ModuleConnectedLivingSpace
    passable = true
  }
}


@PART[kerbalism-gravityring]:HAS[!MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]
{
  MODULE
  {
    name = ModuleConnectedLivingSpace
    passable = true
  }
}

@PART[kerbalism-container-inline-125]:HAS[!MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]
{
  MODULE
  {
    name = ModuleConnectedLivingSpace
    passable = true
  }
}

@PART[kerbalism-container-inline-250]:HAS[!MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]
{
  MODULE
  {
    name = ModuleConnectedLivingSpace
    passable = true
  }
}

@PART[kerbalism-container-inline-375]:HAS[!MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]
{
  MODULE
  {
    name = ModuleConnectedLivingSpace
    passable = true
  }
}
```

**Comment by PiezPiedPy** [comment]
Thanks

**Comment by Gordon-Dry** [comment]
You could let the 125 out because the texture is not intended to be a passable part - but IMHO 1.25m is enough so the hole should be barely wide enough for a Kerbal head ... without helmet haha.

**Comment by PiezPiedPy** [comment]
Modified CLS.cfg to

```
@PART[kerbalism-greenhouse,kerbalism-gravityring,kerbalism-container-inline-*]:HAS[!MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]
{
  MODULE
  {
    name = ModuleConnectedLivingSpace
    passable = true
  }
}

@PART[kerbalism-container-inline-0625]:HAS[MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]
{
  @MODULE[ModuleConnectedLivingSpace]
  {
    @passable = false
  }
}
```

**Comment by PiezPiedPy** [comment]
I'm thinking to completely remove the module from the 0625 container, any objections?

**Comment by Gordon-Dry** [comment]
Well, the 0625 container got no CLS config, so it's okay.

**Comment by Gordon-Dry** [comment]
This
```
@PART[kerbalism-greenhouse,kerbalism-gravityring,kerbalism-container-inline-*]:HAS[!MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]
{
  MODULE
  {
    name = ModuleConnectedLivingSpace
    passable = true
  }
}

@PART[kerbalism-container-inline-0625]:HAS[MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]
{
  !MODULE[ModuleConnectedLivingSpace],* {}
} 
```
is not fully working, because the 0625 part still got CLS passability.

**Comment by PiezPiedPy** [comment]
ahhh, small type on line
`@PART[kerbalism-container-inline-0625]:HAS[MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]`

should be
`@PART[kerbalism-container-inline-0625]:HAS[@MODULE[ModuleConnectedLivingSpace]]:NEEDS[ConnectedLivingSpace]`

**Comment by PiezPiedPy** [comment]
added a commit to fix this in PR #48

**Comment by Gordon-Dry** [comment]
Okay, thanks.
